docs/parameter_tables/HomSap/AncientEurope_4P21.csv

```diff
@@ -0,0 +1,25 @@
+Population size,"50,000",Bronze Age pop. size
+Population size,"5,000",Yamnaya pop. size
+Population size,"10,000",Western Hunter-Gatherer pop. size
+Population size,"10,000,Eastern Hunter-Gatherer pop. size
```

Comment:
missing a closing parenthesis here (I think this is the cause of the CI failure)